### PR TITLE
Updating message for validation timeout and increasing timeout period…

### PR DESCRIPTION
**Pull request by vishalkiri**
Updating message for validation timeout and increasing timeout period for private extensions

### CODE REVIEW

**Comment by vishalkiri** [comment]
@trevorsg Request you to please review

File: app/exec/extension/_lib/publish.ts [hunk 1]
```diff
@@ -388,10 +390,14 @@ export class PackagePublisher extends GalleryBase {
...
         retries: number,
         showPatienceMessageAt: number,
+        publisher: string,
+        extensionId: string,
         version?: string,
     ): Promise<string> {
         if (retries === 0) {
-            throw new Error("Validation timed out. There may be a problem validating your extension. Please try again later.");
+			const validationTimedOutMessage = `Validation is taking much longer than usual. TFX is exiting. To get the validation status, you may run the command below. This extension will be available after validation is successful.\n\n
+                tfx extension isvalid --publisher ${publisher} --extension-id ${extensionId} --version ${version} --service-url ${this.settings.galleryUrl} --token <your PAT>`;
```
Comment (trevorsg) [inline app/exec/extension/_lib/publish.ts]: is there supposed to be 3 new lines here? You have `\n\n` as well as a new line character in the string template literal, which would result in 3 empty lines. I would prefer you stick to using literal newline characters or escaped new lines, not a combination of the two.

Comment (trevorsg) [reply app/exec/extension/_lib/publish.ts]: Also, please use colors.yellow() to highlight the command. See line 315 for an example.

Comment (trevorsg) [reply app/exec/extension/_lib/publish.ts]: Oh, and please fix code formatting.

Comment (vishalkiri) [reply app/exec/extension/_lib/publish.ts]: Done

**Comment by vishalkiri** [comment]
@trevorsg Updated the changes as per suggestions

File: app/exec/extension/_lib/publish.ts [hunk 2]
```diff
@@ -312,19 +312,18 @@ export class PackagePublisher extends GalleryBase {
...
-                if (extInfo.isPublicExtension && this.settings.noWaitValidation) {
-                    trace.info(publicValidationMessage);
+                if (this.settings.noWaitValidation) {
+                    trace.info(validationMessage);
                     return ext;
                 } else {
-                    const validaitonInProgressMessage = extInfo.isPublicExtension
-                        ? publicValidationMessage
-                        : "Validations are in progress. This should take only a few seconds, but in some cases may take a bit longer.";
+                    const validaitonInProgressMessage = validationMessage;
```
Comment (trevorsg) [inline app/exec/extension/_lib/publish.ts]: what is the point of this variable?

**Pull request merged**